tests/providers/amazon/aws/sensors/test_emr_containers.py

```diff
@@ -81,6 +81,4 @@ def test_sensor_defer(self, mock_poke):
         mock_poke.return_value = False
         with pytest.raises(TaskDeferred) as exc:
             self.sensor.execute(context=None)
-        assert isinstance(
-            exc.value.trigger, EmrContainerSensorTrigger
-        ), "Trigger is not a EmrContainerSensorTrigger"
+        assert isinstance(exc.value.trigger, EmrContainerTrigger), "Trigger is not a EmrContainerTrigger"
```

Comment:
Perhaps we can enhance the message by including the trigger in the print.

```suggestion
        assert isinstance(, EmrContainerTrigger), f"{exc.value.trigger} is not a EmrContainerTrigger"
```